src/Microsoft.AspNetCore.Mvc.ViewFeatures/ModelExpressionProvider.cs

```diff
@@ -63,5 +63,30 @@ public class ModelExpressionProvider : IModelExpressionProvider
 
             return new ModelExpression(name, modelExplorer);
         }
+
+        /// <inheritdoc />
+        public ModelExpression CreateModelExpression<TModel>(
```

Comment:
@pranavkm Is it a replacement for ExpressionMetadataProvider.FromStringExpression()? If so, how can I access to it while `ModelExpressionProvider` is internal?

Reply:
The interface `IModelExpressionProvider` is a wrapper for `ExpressionMetadataProvider`. It returns an instance `ModelExplorer` which lets you get to the `ModelMetadata` of the type and the graph of it's properties.